tests/x11/yast2_snapper.pm

```diff
@@ -51,8 +51,13 @@ sub clean_and_quit() {
     if (!$ret) {
         $self->{mute_post_fail} = 1;
         record_soft_failure 'bsc#1032831';
-        $self->failure_analysis();
-        assert_screen 'root-gnome-terminal', 90;
+        # Wait another 90 seconds for the commands to finish
+        my $ret = check_screen('root-gnome-terminal', 90);
```

Comment:
Please don't use check_screen with timeout higher than default as this will waste time in case this is not hit. Better use multi-tag assert_screen and `if match_has_tag`